packages/actor-query-source-identify-hypermedia-sparql/lib/QuerySourceSparql.ts

```diff
@@ -180,10 +190,17 @@ export class QuerySourceSparql implements IQuerySource {
         }));
 
         const cachedCardinality = this.cache?.get(countQuery);
-        if (cachedCardinality !== undefined) {
```

Comment:
Please restore this to what it was. This is needed because cardinalities can be zero.

Reply:
The `cachedCardinality` value is an `RDF.QueryResultCardinality` object or `undefined`, so the new type of checking should also work. I guess it was previously a number, which is why the check was implemented that way.